extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ConfigurationService.java

```diff
@@ -328,20 +273,40 @@ public String getUserSearchFilter() throws GuacamoleException {
     }
 
     /**
-     * Returns the maximum number of seconds to wait for LDAP operations.
+     * Returns the time limit, in seconds, that should be used when
+     * performing LDAP operations.  If no limit is specified a default
+     * of 30 seconds is returned.
      *
      * @return
-     *     The maximum number of seconds to wait for LDAP operations
-     *     as configured in guacamole.properties.
+     *     The number of seconds to allow LDAP operations to run
+     *     before terminating the operation.
      *
      * @throws GuacamoleException
      *     If guacamole.properties cannot be parsed.
      */
-    public int getOperationTimeout() throws GuacamoleException {
+    public Integer getOperationTimeout() throws GuacamoleException {
         return environment.getProperty(
             LDAPGuacamoleProperties.LDAP_OPERATION_TIMEOUT,
             30
         );
     }
 
+    /**
+     * Returns a boolean value that determines whether or not LDAP
+     * search operations follow referrals in the tree.
+     *
+     * @return
+     *     True if LDAP referrals will be followed; false if not.
+     *     Default is false.
+     *
+     * @throws GuacamoleException
+     *     If guacamole.properties cannot be parsed.
+     */
+    public Boolean getFollowReferrals() throws GuacamoleException {
```

Comment:
The comment documenting this function seems improved (more detailed), but why has it been reordered relative to the other functions here?

Reply:
I don't think there's any particular reason for reordering it - I think, as with some of the other stuff you mentioned earlier, the reordering happened when rebasing these changes onto the changes for -243 (following referrals).  Let me see if I can fix this one...

Reply:
Fixed.